src/modules/previewpane/MarkDownPreviewHandler/HTMLParsingExtension.cs

```diff
@@ -0,0 +1,93 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+using Markdig;
+using Markdig.Extensions.Figures;
+using Markdig.Extensions.Tables;
+using Markdig.Renderers;
+using Markdig.Renderers.Html;
+using Markdig.Syntax;
+using Markdig.Syntax.Inlines;
+
+namespace MarkdownPreviewHandler
+{
+    /// <summary>
+    /// Markdig Extension to process html nodes in markdown AST.
+    /// </summary>
+    public class HTMLParsingExtension : IMarkdownExtension
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="HTMLParsingExtension"/> class.
+        /// </summary>
+        public HTMLParsingExtension(string baseUrl = "")
+        {
+            this.BaseUrl = baseUrl;
+        }
+
+        /// <summary>
+        /// Gets or sets path to directory containing markdown file.
+        /// </summary>
+        public string BaseUrl { get; set; }
+
+        /// <inheritdoc/>
+        public void Setup(MarkdownPipelineBuilder pipeline)
+        {
+            // Make sure we don't have a delegate twice
+            pipeline.DocumentProcessed -= this.PipelineOnDocumentProcessed;
+            pipeline.DocumentProcessed += this.PipelineOnDocumentProcessed;
+        }
+
+        /// <inheritdoc/>
+        public void Setup(MarkdownPipeline pipeline, IMarkdownRenderer renderer)

```

Comment:
Just to clarify, is this intended to be an empty definition? Can you add the summary tags for the Setup functions as well since I'm not sure what's it used for.

Reply:
It is empty if extension is renderer independent.

Reply:
Can you add the summary comments for those two functions? Or is it there in some other file that I'm missing?